deploy/external-secrets-env.properties

```diff
@@ -0,0 +1,9 @@
+# kubectl -n=es-kubectl create configmap external-secrets-config --from-env-file=deploy/external-secrets-env.properties
```

Comment:
Should we commit the `ConfigMap` instead of the env file that we then create a `ConfigMap` with?  I'd also include the `ConfigMag` and `Deployment` in one yaml file.

Reply:
I don't feel strongly about putting them in one file, but one advantage of them being separate is that it makes it easier to update the configmap separately if you need to change it

Reply:
Actually, since it's a sample, we could provide both